pom.xml

```diff
@@ -50,10 +50,15 @@
         </pluginRepositories>
 
 	<dependencies>
-                <dependency>
-                    <groupId>org.rascalmpl</groupId>
-                    <artifactId>rascal</artifactId>
-                    <version>0.24.4</version>
-                </dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.rascalmpl</groupId>
+            <artifactId>rascal</artifactId>
+            <version>0.24.4</version>
+        </dependency>
+        <dependency>
+            <groupId>iguana</groupId>
```

Comment:
To get the Iguana dependency, clone the Iguana project and run `mvn clean install`. That would put the jar into the local maven repo.